### PR TITLE
fix(feature media): Fixed issue with single line input

### PR DESCRIPTION
**Pull request by pavlovicnemanja**

### CODE REVIEW

File: scripts/apps/authoring/views/item-association.html [hunk 1]
```diff
@@ -15,14 +15,14 @@
...
-        ng-model-options="{debounce: 1000}"
-        ng-change="onchange()"
-        contenteditable="true"
-        ng-if="related"
-        class="image-caption contenteditable-input">
-    </div>
+    <textarea sd-remove-tags
+              sd-auto-height
+              tansa-scope-sync
+              type="text"
```
Comment (petrjasek) [inline scripts/apps/authoring/views/item-association.html]: imo there is no type

File: scripts/core/editor2/views/block-embed.html [hunk 2]
```diff
@@ -13,14 +13,15 @@
...
-        ng-model="vm.caption"
-        ng-model-options="{debounce: 1000}"
-        ng-change="vm.saveCaption(vm.caption)"
-        ng-paste="vm.handlePaste($event)"
-        class="image-caption contenteditable-input">
+    <textarea sd-remove-tags
+              sd-auto-height
+              tansa-scope-sync
+              placeholder="{{::'Description'|translate}}"
+              type="text"
```
Comment (petrjasek) [inline scripts/core/editor2/views/block-embed.html]: same here

File: scripts/apps/authoring/views/item-association.html (hunk 1 above)
Comment (pavlovicnemanja) [inline scripts/apps/authoring/views/item-association.html]: sorry, forget to remove

File: scripts/core/editor2/views/block-embed.html (hunk 2 above)
Comment (pavlovicnemanja) [inline scripts/core/editor2/views/block-embed.html]: same here 😄

**Pull request merged**